python/cuml/cuml/tests/dask/test_dask_logistic_regression.py

```diff
@@ -427,31 +426,29 @@ def assert_small(X, y, n_classes):
 
 
 @pytest.mark.parametrize("n_parts", [2, 23])
-@pytest.mark.parametrize("fit_intercept", [False, True])
-@pytest.mark.parametrize("n_classes", [8])
-def test_n_classes(n_parts, fit_intercept, n_classes, client):
-    datatype = np.float32 if fit_intercept else np.float64
+@pytest.mark.parametrize("n_classes", [2, 6])
+def test_n_classes(n_parts, n_classes, client):
     nrows = int(1e5) if n_classes < 5 else int(2e5)
     lr = _test_lbfgs(
         nrows=nrows,
         ncols=20,
         n_parts=n_parts,
-        fit_intercept=fit_intercept,
-        datatype=datatype,
+        fit_intercept=True,
```

Comment:
Not a big deal, but was wondering if we should use `fit_intercept=True` and `fit_intercept=False` in the tests, so not testing both in each test, but making sure that we have some tests for each value. But yeah 🤷

Reply:
Like with `n_classes` above, we already have places where we check both intentionally for checking intercept behavior. I'd rather be consistent and use sklearn's defaults in places where we aren't explicitly checking a feature.